tutorial_sarscov2.md

```diff
@@ -0,0 +1,259 @@
+  we expect that ShoRAH will consider the variants of poor quality and reject them
+  because there is very little support ( \<=  than 5 reads supporting the alt).
+
+## Swapping component
+
+The default configuration uses [ShoRAH](https://cbg-ethz.github.io/shorah/)
+to call the SNVs and to reconstruct the local (windowed) haplotypes.
+
+Components can be swapped simply by changing the `vpipe.config` file.
+For example to call SNVs using `lofreq`:
+
+```
+[output]
+snv = True
+local = False
+
+[general]
+snv_caller=lofreq
+```
+
+
+## Cluster deployment
+
+It is possible to ask snakemake to [submit jobs on a cluster](https://snakemake.readthedocs.io/en/stable/executing/cluster-cloud.html#cluster-execution) using the batch submission command-line interface of your cluster.
+
+[Platform LSF by IBM](https://www.ibm.com/support/knowledgecenter/SSETD4_9.1.2/lsf_command_ref/bsub.1.html) is one of the popular systems you might find (Others include [SLURM](https://slurm.schedmd.com/sbatch.html), [Grid Engine](https://en.wikipedia.org/wiki/Oracle_Grid_Engine)).
+
+*[LSF]: Load Sharing Facility
+
+To deploy on the cluster:
+
+```bash
+wget 'https://raw.githubusercontent.com/cbg-ethz/V-pipe/master/deploy_vpipe.sh'
+bash deploy_vpipe.sh -b sars-cov2 -p $SCRATCH -w working
+cd $SCRATCH/working/
+```
+
+ - using `-p` help us store V-pipe on some large-storage share, e.g.: scratch
+ 
+> **Tips:** As V-pipe for SARS-CoV-2 matures, it will be possible to download [snapshots frozen at specific version](https://github.com/cbg-ethz/V-pipe/releases).
+> This enables more reproducible results. To specify a release use the `-r` option :
+```bash
+wget 'https://raw.githubusercontent.com/cbg-ethz/V-pipe/master/deploy_vpipe.sh'
+bash deploy_vpipe.sh -r sars-cov2-snapshot-20200406 -p $SCRATCH -w working
+cd $SCRATCH/working/
+```
+> this will download the tarball [sars-cov2-snapshot-20200406.tar.gz](https://github.com/cbg-ethz/V-pipe/archive/sars-cov2-snapshot-20200406.tar.gz) and uncompress it into a directory called `V-pipe-sars-cov2-snapshot-20200406`
+
+### Running V-pipe on the cluster
+
+In the `working` directory, create a `samples` sub-directory and populate it. Check its structure with `tree` or `find`. Perform the necessary adjustments to `vpipe.config`.
+
+To run V-pipe on a cluster :
+
+> **Tips:** There are [snakemake parameters for conda](https://snakemake.readthedocs.io/en/stable/executing/cli.html#CONDA)
+that can help management of dependencies:
+> - using `--create-envs-only` enables to download the dependencies only without running the pipeline itself.
+> - using `--conda-prefix {DIR}`  stores the conda environments of dependencies in a common directory (thus possible to share re-use between multiple instances of V-pipe).
+
+```bash
+# Download everything in advance
+./vpipe --conda-prefix $SCRATCH/snake-envs --cores all --create-envs-only
+
+# Cluster LSF submitting
+./vpipe --conda-prefix $SCRATCH/snake-envs -p --cluster 'bsub' --jobs 2
```

Comment:
We should not run the master job on the login node (add bsub) and point to  https://github.com/cbg-ethz/V-pipe/wiki/advanced#running-v-pipe-on-a-lsf-cluster